src/node-client/lib/Convex/NodeClient/Progress.hs

```diff
@@ -1,8 +1,10 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE LambdaCase #-}
-{-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE NumericUnderscores #-}
+-- FIXME (j-mueller) The deprecation warning on Cardano.Api.Block covers the whole type instead of just the view pattern 'Block' that seems to have been intended.
```

Comment:
> that seems to have been intended.

You meant "was *not* intended"?

Reply:
I meant that the intention was to only mark the view pattern as deprecetad. I'll rephrase this